client/state/sites/selectors/is-jetpack-cloud-cart-enabled.js

```diff
@@ -0,0 +1,15 @@
+import { isEnabled } from '@automattic/calypso-config';
+import { isUserLoggedIn } from 'calypso/state/current-user/selectors';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import getSiteSlug from './get-site-slug';
+
+/**
+ * Returns true if we want to show Cart in Jetpack cloud
+ *
+ * @param  {object}   state  Global state tree
+ */
+export default function isJetpackCloudCartEnabled( state ) {
+	const siteId = getSelectedSiteId( state );
+	const siteSlug = getSiteSlug( state, siteId );
+	return isEnabled( 'jetpack/pricing-page-cart' ) && isUserLoggedIn( state ) && siteSlug;
```

Comment:
Really minor and probably not worth it but just so you know, this line will actually return a string (or false) and since you probably want to return a boolean always, it would be a good idea to coerce the type to a boolean. (This could also be done when this file is converted to TS).
```suggestion
	return Boolean( isEnabled( 'jetpack/pricing-page-cart' ) && isUserLoggedIn( state ) && siteSlug );
```